app/models/manageiq/providers/nuage/manager_mixin.rb

```diff
@@ -4,45 +4,140 @@ module ManageIQ::Providers::Nuage::ManagerMixin
+                        :label      => _("Security Protocol"),
+                        :isRequired => true,
+                        :validate   => [{:type => "required-validator"}],
+                        :options    => [
+                          {
+                            :label => _("SSL without validation"),
+                            :value => "ssl-no-validation"
+                          },
+                          {
+                            :label => _("SSL"),
+                            :value => "ssl-with-validation"
+                          },
+                          {
+                            :label => _("Non-SSL"),
+                            :value => "non-ssl"
+                          }
+                        ]
+                      },
+                      {
+                        :component  => "text-field",
+                        :name       => "endpoints.default.default_hostname",
+                        :label      => _("Hostname (or IPv4 or IPv6 address)"),
+                        :isRequired => true,
+                        :validate   => [{:type => "required-validator"}],
+                      },
+                      {
+                        :component    => "text-field",
+                        :name         => "endpoints.default.port",
+                        :label        => _("API Port"),
+                        :type         => "number",
+                        :initialValue => 443,
+                        :isRequired   => true,
+                        :validate     => [{:type => "required-validator"}],
+                      },
+                      {
+                        :component  => "text-field",
+                        :name       => "authentications.default.userid",
+                        :label      => "Username",
+                        :isRequired => true,
+                        :validate   => [{:type => "required-validator"}],
+                      },
+                      {
+                        :component  => "text-field",
+                        :name       => "authentications.default.password",
+                        :label      => "Password",
+                        :type       => "password",
+                        :isRequired => true,
+                        :validate   => [{:type => "required-validator"}],
+                      }
+                    ]
+                  }
+                ]
+              },
+              {
+                :component => 'tab-item',
+                :name      => 'events',
+                :title     => _('Events'),
+                :fields    => [
+                  {
+                    :component    => 'select-field',
+                    :name         => 'event_stream_selection',
+                    :initialValue => 'amqp',
+                    :label        => _('Type'),
+                    :options      => [
```

Comment:
I think this need `None` as an option and then it should be the default

Reply:
:+1: added

Reply:
@skateman Is there a more general way to say "event creds are not required"?

Reply:
Not really, just have the `none` with an empty string as the value